### PR TITLE
changed github.com/pingcap/ticdc to github.com/pingcap/tiflow

### PR DESCRIPTION
**Pull request by hfxsd**
changed github.com/pingcap/ticdc to github.com/pingcap/tiflow

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/configure-a-tidb-cluster.md [hunk 1]
```diff
@@ -433,7 +433,7 @@ spec:
       logLevel: info
 ```
 
-For all configurable start parameters of TiCDC, see [TiCDC configuration](https://github.com/pingcap/ticdc/blob/master/cmd/ticdc.toml).
+For all configurable start parameters of TiCDC, see [TiCDC configuration](https://github.com/pingcap/tiflow/blob/master/cmd/ticdc.toml).
```
Comment (csuzhangxc) [inline en/configure-a-tidb-cluster.md:436]: 404 reported for `https://github.com/pingcap/tiflow`, it's a private repo?

Comment (ran-huang) [reply en/configure-a-tidb-cluster.md:436]: Same 404 here. Please do not merge until the repo is publicly accessible.

**Pull request merged**